src/lib/serviceRoutinesV2/getEntities.cpp

```diff
@@ -129,27 +130,27 @@ std::string getEntities
   //
   if ((coords != "") && (geometry == ""))
   {
-    OrionError   oe(SccInvalidModification, "Query not supported: URI param /coords/ used without /geometry/");
+    OrionError oe(SccBadRequest, "Wrong query: URI param /coords/ used without /geometry/", "BadRequest");
```

Comment:
Wrong => Invalid

Wrong ... means 'not the right one', but it says nothing about whether it is valid or not. We should always use 'Invalid' for this kind of situations.